Validation/RecoVertex/python/HLTmultiPVvalidator_cff.py

```diff
@@ -2,46 +2,46 @@
-hltMultiPVanalysis.trackAssociatorMap    = cms.untracked.InputTag("trackingParticleRecoTrackAsssociation")
-hltMultiPVanalysis.vertexAssociator      = cms.untracked.InputTag("VertexAssociatorByPositionAndTracks")
-
+hltMultiPVanalysis = vertexAnalysis.clone(
+do_generic_sim_plots  = False,
+verbose               = False,
+root_folder           = "HLT/Vertexing/ValidationWRTsim",
+vertexRecoCollections = ("",),
+trackAssociatorMap    = "trackingParticleRecoTrackAsssociation",
+vertexAssociator      = "VertexAssociatorByPositionAndTracks"
+)
 from Validation.RecoTrack.associators_cff import hltTrackAssociatorByHits, tpToHLTpixelTrackAssociation
 from SimTracker.VertexAssociation.VertexAssociatorByPositionAndTracks_cfi import VertexAssociatorByPositionAndTracks as _VertexAssociatorByPositionAndTracks
-vertexAssociatorByPositionAndTracks4pixelTracks = _VertexAssociatorByPositionAndTracks.clone()
-vertexAssociatorByPositionAndTracks4pixelTracks.trackAssociation = cms.InputTag("tpToHLTpixelTrackAssociation")
-
+vertexAssociatorByPositionAndTracks4pixelTracks = _VertexAssociatorByPositionAndTracks.clone(
+trackAssociation = "tpToHLTpixelTrackAssociation"
+)
 tpToHLTpfMuonMergingTrackAssociation = tpToHLTpixelTrackAssociation.clone(
     label_tr = "hltPFMuonMerging"
 )
-vertexAssociatorByPositionAndTracks4pfMuonMergingTracks = _VertexAssociatorByPositionAndTracks.clone()
-vertexAssociatorByPositionAndTracks4pfMuonMergingTracks.trackAssociation = cms.InputTag("tpToHLTpfMuonMergingTrackAssociation")
-
+vertexAssociatorByPositionAndTracks4pfMuonMergingTracks = _VertexAssociatorByPositionAndTracks.clone(
+trackAssociation = "tpToHLTpfMuonMergingTrackAssociation"
+)
 
 
-hltPixelPVanalysis = hltMultiPVanalysis.clone()
-hltPixelPVanalysis.do_generic_sim_plots  = True
-hltPixelPVanalysis.trackAssociatorMap    = cms.untracked.InputTag("tpToHLTpixelTrackAssociation")
-hltPixelPVanalysis.vertexAssociator      = cms.untracked.InputTag("vertexAssociatorByPositionAndTracks4pixelTracks")
-hltPixelPVanalysis.vertexRecoCollections = cms.VInputTag(
+hltPixelPVanalysis = hltMultiPVanalysis.clone(
+do_generic_sim_plots  = True,
+trackAssociatorMap    = "tpToHLTpixelTrackAssociation",
+vertexAssociator      = "vertexAssociatorByPositionAndTracks4pixelTracks",
+vertexRecoCollections = (
     "hltPixelVertices",
     "hltTrimmedPixelVertices",
 )
+)
 
 
-
-hltPVanalysis = hltMultiPVanalysis.clone()
-hltPVanalysis.trackAssociatorMap = cms.untracked.InputTag("tpToHLTpfMuonMergingTrackAssociation")
-hltPVanalysis.vertexAssociator   = cms.untracked.InputTag("vertexAssociatorByPositionAndTracks4pfMuonMergingTracks")
-hltPVanalysis.vertexRecoCollections   = cms.VInputTag(
-    "hltVerticesPFFilter"
+hltPVanalysis = hltMultiPVanalysis.clone(
+trackAssociatorMap = "tpToHLTpfMuonMergingTrackAssociation",
+vertexAssociator   = "vertexAssociatorByPositionAndTracks4pfMuonMergingTracks",
+vertexRecoCollections   = (
+    "hltVerticesPFFilter",
 #    "hltFastPVPixelVertices"
 )
```

Comment:
```suggestion
vertexRecoCollections   = ["hltVerticesPFFilter"]   #"hltFastPVPixelVertices"
```
(or, even better, remove the commented out part if not needed any more)